### PR TITLE
updating distro parameters

### PR DESCRIPTION
**Pull request by pondrejk**
Getting the ui jobinvocation tests on par with ui remoteexecution tests. The vm_host is now reused, the distro was updated from rhel 6. 

Test result (standalone automation job no 1821)
```py.test -v --junit-xml=foreman-results.xml -o junit_suite_name=standalone-automation -m 'not stubbed' tests/foreman/ui/test_jobinvocation.py

============================= test session starts ==============================
...
collected 2 items

tests/foreman/ui/test_jobinvocation.py::test_positive_run_default_job_template_by_ip PASSED [ 50%]
tests/foreman/ui/test_jobinvocation.py::test_positive_run_custom_job_template_by_ip PASSED [100%]

==================== 2 passed, 1 warnings in 406.83 seconds ====================
```

### CODE REVIEW

File: tests/foreman/ui/test_jobinvocation.py [hunk 1]
```diff
@@ -18,41 +18,69 @@
...
 from robottelo.helpers import add_remote_execution_ssh_key
 from robottelo.vm import VirtualMachine
 
 
+def _setup_vm_client_host(vm_client, org_label, subnet_id=None, by_ip=True):
+    """Setup a VM host for remote execution.
+
+    :param VirtualMachine vm_client: where vm_client is VirtualMachine instance.
+    :param str org_label: The organization label.
+    :param int subnet: (Optioanl) Nailgun subnet entity id, to be used by the vm_client host.
```
Comment (mirekdlugosz) [inline tests/foreman/ui/test_jobinvocation.py]: Typo:
```suggestion
    :param int subnet: (Optional) Nailgun subnet entity id, to be used by the vm_client host.
```

**Comment by pondrejk** [comment]
@mirekdlugosz I fixed both occurrences of the typo. Putting the function up would certainly make sense, if I may I'd reserve it for a separate commit, I'd like to see first if there are also other test classes that could consume it

**Comment by pondrejk** [comment]
@omkarkhatavkar @mirekdlugosz since there are two acks in place, can I humbly ask for merge?

**Comment by mirekdlugosz** [comment]
Just to explain the situation:

According to "PR Review Process" (Mojo DOC-1187945), PR needs at least 1 ACK from Tier2 reviewer before merge. Me and Omkar are Tier1, so it was waiting for Tier2 to take a look.

Yes, according to the same doc, it is Tier2 reviewers job to "ensure the PR is resolved quickly". One would think two weeks are more than enough.

**Pull request merged**